linux/test/app.js

```diff
@@ -0,0 +1,31 @@
+const Application = require('spectron').Application;
```

Comment:
Can use de construct const { Application } = require('spectron')

Reply:
Also, The same test file is not specific to Linux and can be done on all environments. How about we move it to the root as well (with all the packages). You don't need to change the content except the appPath and I'll make sure it will be part win and Mac builds as well